### ISSUE TITLE
Add links to the blog and examples.pyviz.org in the navigation bar

### ISSUE THREAD

**Issue by maximlt** [holoviz/holoviz#334]
I just thought about it and these are easy wins in my opinion. The blog is almost unknown to all and that would expose it more. There's currently a `Topics` page that links to examples.pyviz.org, EarthML and EarthSim. I'm not sure we really want people to dig into EarthML and EarthSim as these projects aren't maintained (it's hard for anyone right now to run their code, due to dependencies drifting), while examples.pyviz.org is (a little more) maintained and should contain reproducible examples. It's also meant to move and be added as a holoviz.org subdomain, so we can already make it closer to holoviz.org.

The only issue I see is that it would clutter the navigation bar. I think we could remove the `Introduction` link as you get the same behavior by clicking on the logo (that is a standard UX on the web, I think) and I think there's a plan to move the `Installation` link as it's going to be embedded in the `Tutorial`. `Talks` is a little bit of a weird one as it contains the slides of a single presentation but together in a page. `Roadmap`, `FAQ`, `About` all seem somewhat related. I think on a traditional site these links would go into a footer. Maybe that's something to think about?

It might be a good idea to think about how this site could be structured, if we were to start it from scratch today. While the other HoloViz sites are usually dedicated to documenting using a software, this one feels more like it's showing off a product and the community that built it.

**Comment by droumis** [comment]
Indeed, we spoke a bit about a new top-level nav organization for HoloViz.org during a meeting about a month ago.

Here is what was proposed, which seemed at the time to reach a consensus: 

## Current Nav:
- Introduction
- Background
- Installation
- Talks
- Tutorial
- Topics
- Roadmap
- FAQ
- About
- Community

## Proposed Nav:
- Home
- Get Started
- Tutorials
- Topic Gallery
- Contribute
- FAQ
- About
- Community

I could be convinced to add a 'Blog' to this if we improved the blog appearance :)

I think the current examples.pyviz.org should become a part of the HoloViz.org > Topic Gallery

**Comment by maximlt** [comment]
Well first you add a link to the blog, and then you improve the blog because you feel ashamed of how it looks. That's the only way to get it done :D

I think examples.pyviz.org will keep on living in its own repo, as it's pretty complex to maintain it.

**Comment by jbednar** [comment]
I think we should go ahead with the move examples.pyviz.org -> examples.holoviz.org and then simply have the Topics or Topic Gallery or Examples send the user there. And sure, add a link to the blog!
